content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Resource Types
+---
```

Comment:
```suggestion
---
title: Resource Types
content_type: reference
---
```

Reply:
Where can I find the list of `content_type`s?